source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -303,11 +409,19 @@ void apply_impact(boost::shared_ptr<nt::new_disruption::Impact>impact,
     boost::for_each(impact->informed_entities, boost::apply_visitor(v));
 }
 
+template<typename T>
+void reindex(std::vector<T> vec) {
```

Comment:
const& for the vector

ok it does not change anything, but for the record there is somethink in utils to do that:

```boost::for_each(collection_name, Indexer<nt::idx_t>());```